solr/bin/solr.cmd

```diff
@@ -263,7 +263,6 @@ IF "%1"=="assert" goto run_solrcli
 IF "%1"=="zk" goto run_solrcli
```

Comment:
Maybe for another jira, but the way `bin/solr` works is that all the commands go thorugh a check for if you are special..  For example, here is the auth check: https://github.com/apache/solr/blob/main/solr/bin/solr#L594 and then if you aren't start/stop/restart, you get the `run_solrcli` treatment: https://github.com/apache/solr/blob/main/solr/bin/solr#L629.   This removed a bunch of lines!

Reply:
I am looking into this right now. We have a lot of repetitive code in `bin/solr.cmd` that can be optimized if code blocks are rearranged. The way `bin/solr` is handling the special commands is probably the approach we gonna follow on the windows script as well.